std-bits/table/src/main/java/org/enso/table/aggregations/CountDistinct.java

```diff
@@ -0,0 +1,41 @@
+package org.enso.table.aggregations;
+
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.index.MultiValueKey;
+import org.enso.table.data.table.Column;
+import org.enso.table.data.table.problems.FloatingPointGrouping;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/***
+ * Aggregate Column counting the number of distinct items in a group.
+ */
+public class CountDistinct extends Aggregator {
+  private final Storage[] storage;
+  private final boolean ignoreEmpty;
+
+  public CountDistinct(String name, Column[] columns, boolean ignoreEmpty) {
```

Comment:
I think `ignoreEmpty` should be documented to indicate what does it mean.

Also shouldn't it be `ignoreAllNulls`? Empty has a different meaning in these functions than what you have here which is a potential source of confusion